openpype/hosts/maya/api/workfile_template_builder.py

```diff
@@ -244,13 +233,36 @@ def get_placeholder_options(self, options=None):
         return self.get_load_plugin_options(options)
 
     def post_placeholder_process(self, placeholder, failed):
-        """Hide placeholder, add them to placeholder set
-        """
+        """Hide placeholder, add them to placeholder set"""
         node = placeholder.scene_identifier
 
         cmds.sets(node, addElement=PLACEHOLDER_SET)
         cmds.hide(node)
         cmds.setAttr(node + ".hiddenInOutliner", True)
+        model = cmds.ls(assemblies=True)
+
+        filtered_model = []
+        exclude_list = ["persp", "top", "front", "side", "root", "wip"]
+
+        for element in model:
+            if not any(exclude_elem in element for exclude_elem in exclude_list):
+                filtered_model.append(element)
+
+        cameras = cmds.ls(type="camera")
+        renderable_camera = None
+
+        for camera in cameras:
+            if cmds.getAttr(camera + ".renderable"):
+                shapes = cmds.listRelatives(
+                    camera, parent=True, fullPath=True, type="transform"
+                )
+                if shapes:
+                    renderable_camera = shapes[0]
+                    break
+
+        cmds.lookThru(renderable_camera)
+        print("filtered_model: ", filtered_model)
+        cmds.viewFit(filtered_model)
```

Comment:
I'd argue that it doesn't make sense to do this in `post_placeholder_process` if that's called PER placeholder - since we'd be doing this many many times. We should be doing this at most ONCE after the whole build process completed - and better yet, I'd even argue that this shouldn't be in the default template logic - this should be something that can be added to a build process separately instead of forced upon.

Reply:
We will add this specifically as action, it won't stay in part of the code. So as you say, it will be called per placeholder as you say.